textgrad/engine/__init__.py

```diff
@@ -5,6 +5,7 @@
     "haiku": "claude-3-haiku-20240307",
     "sonnet": "claude-3-sonnet-20240229",
     "together-llama-3-70b": "together-meta-llama/Llama-3-70b-chat-hf",
+    "qwen2": "qwen2:0.5b",
```

Comment:
it may be good to name this w/ the provider and the model size, in case we want to have other qwen models in the future. 

i.e., something like `ollama-qwen2-0.5b` or something, in case we later want to add other sizes / providers for qwen.